packages/mobile/src/invite/saga.ts

```diff
@@ -377,17 +394,17 @@ export function* skipInvite() {
   }
 }
 
-function* addTempAccountToWallet(inviteCode: string) {
+function* addTempAccountToWallet(tempAccountPrivateKey: string) {
   Logger.debug(TAG + '@addTempAccountToWallet', 'Attempting to add temp wallet')
   try {
     // Import account into the local geth node
     const wallet: UnlockableWallet = yield call(getWallet)
-    const account = privateKeyToAddress(inviteCode)
+    const account = privateKeyToAddress(tempAccountPrivateKey)
     const password: string = yield call(getPasswordSaga, account, false, true)
-    const tempAccount = yield call([wallet, wallet.addAccount], inviteCode, password)
+    const tempAccount = yield call([wallet, wallet.addAccount], tempAccountPrivateKey, password)
     Logger.debug(TAG + '@addTempAccountToWallet', 'Account added', tempAccount)
   } catch (e) {
-    if (e.toString().includes('account already exists')) {
+    if (e.message === ErrorMessages.GETH_ACCOUNT_ALREADY_EXISTS) {
```

Comment:
I think the error message may have changed with recent changes to geth. What I've done seems to produce the desired outcome